# Goal
string target_map
geometry_msgs/PoseStamped goal_pose
---
# Result
bool success
---
# Feedback
string current_step

